client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -1263,8 +1272,8 @@ const Editor = (props: EditorProps): JSX.Element => {
     theme === Themes.Night
       ? 'vs-dark-custom'
       : theme === Themes.Default
-        ? 'vs-custom'
-        : editorSystemTheme;
+      ? 'vs-custom'
+      : editorSystemTheme;
```

Comment:
Apparently prettier thinks these should all line up. I'm actually a little surprised we have nested ternaries here. But if we are going to keep them then I think it makes sense to nest the second one. I'll put it back.

Reply:
Well, apparently I need some guidance here, because every time I put the original indentation back on the second ternary and then do a `git commit`, the indentation gets removed by prettier and both ternaries are back to lining up with the same indentation.